### DISCUSSION TITLE
WebSocket connection problem

### DISCUSSION THREAD

**Discussion by marmundo** [mqttjs/MQTT.js#1741] [Q&A]
Hi,

I've tried to connect a broker using self-asigned certificate without sucess. When I use postman, I can connect. WIthout put postman (electron) on it. How can I connect? 
My error is just `index.jsx:58 WebSocket connection to 'wss://10.225.21.3:8091/' failed: `
On client.stream on I log this object:
```js
isTrusted :true
bubbles: false
cancelBubble: false
cancelable:false
composed:false
currentTarget:{WebSocket
binaryType:"arraybuffer"
bufferedAmount:0
extensions:""
onclose
: 
null
onerror
: 
null
onmessage
: 
null
onopen
: 
null
protocol
: 
""
readyState
: 
3
url
: 
"wss://10.225.21.3:8091/"}
```
This is my code
```js
const options = {
    clientId: Math.random().toString(16).substring(2, 8),
    username: brokerUser,
    password: brokerPassword,
    path: "",
    // rejectUnauthorized: false,
  };

  let client = mqtt.connect("wss://10.225.21.3:8091", options);
```

Thanks

**Comment by robertsLando** [comment]
You should open dev tools and check the error there to know more details about why the connection has been rejected
